task.js

```diff
@@ -28,7 +28,17 @@ class Task {
     this.mapper = mapper || function() { return ''; };
     this.translations = [];
   }
+  static async logError(title,error){
```

Comment:
See previous comment. No need to pass `title` as argument. `error` is sufficient.